tests/framework/db/conditions/LikeConditionTest.php

```
@@ -0,0 +1,217 @@
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage("Operator 'LIKE' requires two operands.");
+
+        LikeCondition::fromArrayDefinition('LIKE', ['name']);
+    }
+
+    public function testFromArrayDefinitionThrowsOnEmptyOperands(): void
+    {
+        $this->expectException(InvalidArgumentException::class);
+
+        LikeCondition::fromArrayDefinition('LIKE', []);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\db\conditions\LikeCondition' in method 'testFromArrayDefinitionThrowsOnEmptyOperands'. (undefined)

(StaticAccess)